lib/stoplight/traffic_control/consecutive_failures.rb

```diff
@@ -14,18 +14,30 @@ module TrafficControl
     #    reach the threshold.
     #
     # @example With window-based configuration
-    #   config = Stoplight::Light::Config.new(threshold: 5, window_size: 60)
-    #   strategy = Stoplight::TrafficControlStrategy::ConsecutiveFailures.new
+    #   traffic_control = Stoplight::TrafficControlStrategy::ConsecutiveFailures.new
+    #   config = Stoplight::Light::Config.new(threshold: 5, window_size: 60, traffic_control:)
     #
     # Will switch to red if 5 consecutive failures occur within the 60-second window
     #
     # @example With total number of consecutive failures configuration
-    #   config = Stoplight::Light::Config.new(threshold: 5, window_size: nil)
-    #   strategy = Stoplight::TrafficControlStrategy::ConsecutiveFailures.new
+    #   traffic_control = Stoplight::TrafficControlStrategy::ConsecutiveFailures.new
+    #   config = Stoplight::Light::Config.new(threshold: 5, window_size: nil, traffic_control:)
     #
     # Will switch to red only if 5 consecutive failures occur regardless of the time window
     # @api private
     class ConsecutiveFailures < Base
+      # @param config [Stoplight::Light::Config]
+      # @return [Stoplight::Config::CompatibilityResult]
+      def check_compatibility(config)
+        if config.threshold < 1
```

Comment:
```suggestion
        if config.threshold <= 0
```
I think it's more clear 🤔 WDYT?